src/Versioned.php

```diff
@@ -2797,8 +2844,78 @@ public static function get_including_deleted($class, $filter = "", $sort = "")
         if (!empty($sort)) {
             $list = $list->orderBy($sort);
         }
-        $list = $list->setDataQueryParam("Versioned.mode", "latest_versions");
-        return $list;
+        return static::updateListToIncludeDeleted($list);
+    }
+
+    /**
+     * Update a DataList to query the latest version of each record stored in the (class)_Versions tables.
+     *
+     * In particular, this will query deleted records as well as active ones.
+     */
+    public static function updateListToIncludeDeleted(DataList $list): DataList
+    {
+        return $list->setDataQueryParam('Versioned.mode', 'latest_versions');
+    }
+
+    /**
+     * Return the equivalent of a DataList::create() call, querying only records which have been removed
+     * from draft. This includes archived records and records which were published but have no draft record.
+     *
+     * @template T of DataObject
+     * @param class-string<T> $class
+     * @param string $where Explicitly used in where clause as raw SQL - use with caution!
+     * @param string $orderBy Explicitly used in orderBy clause as raw SQL - use with caution!
+     * @return DataList<T>
+     */
+    public static function getRemovedOnDraft(string $class, string|array $where = '', string $orderBy = ''): DataList
+    {
+        $list = DataList::create($class);
+        if (!empty($where)) {
+            $list = $list->where($where);
+        }
+        if (!empty($orderBy)) {
+            $list = $list->orderBy($orderBy);
+        }
+        return static::updateListToIncludeRemovedOnDraft($list);
+    }
+
+    /**
+     * Update a DataList to query only records which have been removed from draft.
+     * This includes archived records and records which were published but have no draft record.
+     */
+    public static function updateListToIncludeRemovedOnDraft(DataList $list): DataList
```

Comment:
```suggestion
    public static function updateDataListToOnlyIncludeRemovedFromDraft(DataList $list): DataList
```
And update references to this

Just because `removed_on_draft` is a poor name shouldn't mean we need to repeat it, "removed from draft" is far more intuitive

Reply:
Done